src/session_client.c

```diff
@@ -988,6 +995,26 @@ build_module_info_yl(struct nc_session *session, int get_data_sup, int xpath_sup
         }
     }
 
+    /* If NETCONF server supports RFC5277 notification capability and libnetconf2
+     * required notifications and nc-notifications are not present on the NETCONF
+     * server (which it is not obligated to support), then the libyang context
+     * needs to be initialized using client side local YANG schema files */
+    if (nc_session_cpblt(session, "urn:ietf:params:netconf:capability:notification:1.0") &&
+        !notifications_found && !nc_notifications_found) {
```

Comment:
Another formatting error, this line needs one more indent.

Reply:
Resolved in latest push